code/modules/client/preferences_savefile.dm

```diff
@@ -112,6 +112,19 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	if(current_version < 24)
 		if (!(underwear in GLOB.underwear_list))
 			underwear = "Nude"
+	if(current_version < 25)
+		if(S["name_is_always_random"] == 1)
```

Comment:
You need to actually initialize randomise list here. At this point it's null. So you end up with "randombodyrandomnamerandomspecies" string in there at the end of the update.

Reply:
@anturk by initialise do you just mean randomise = list() ?

Reply:
Yep